### PR TITLE
Create HARC-HAKRCF405V2.config

### PR DESCRIPTION
**Pull request by sugaarK**
new HARC target with ICM 42688-p gyro

### CODE REVIEW

File: configs/default/HARC-HAKRCF405ICM.config [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+dma pin B01 0
+# pin B01: DMA1 Stream 2 Channel 5
+dma pin B08 0
+# pin B08: DMA1 Stream 7 Channel 2
+dma pin B03 0
+# pin B03: DMA1 Stream 6 Channel 3
+
+# feature
+feature RX_SERIAL 
+feature OSD
```
Comment (howels) [inline configs/default/HARC-HAKRCF405ICM.config:108]: OSD is user defined in cloud builds.

**Review by howels — CHANGES_REQUESTED** [review]

File: configs/default/HARC-HAKRCF405ICM.config [hunk 2]
```diff
@@ -0,0 +1,129 @@
+# Betaflight / STM32F405 (S405) 4.2.4
+
+#define USE_GYRO_SPI_ICM42688P 
+#define USE_ACC_SPI_ICM42688P
+#define USE_FLASH_W25Q128FV
+#define USE_MAX7456
+
+board_name HAKRCF40ICM
```
Comment (KarateBrot) [inline configs/default/HARC-HAKRCF405ICM.config:8]:
```suggestion
board_name HAKRCF405ICM
```

Are you sure this is a good board name? If in future there are other gyros on board this name will be very misleading.

**Review by KarateBrot — CHANGES_REQUESTED** [review]

**Comment by sugaarK** [comment]
What would you recommend @KarateBrot ? This came from my contact.. I asked what they wanted as when I got to it I’d be drinking and my go for something witty

File: configs/default/HARC-HAKRCF405ICM.config (hunk 1 above)
Comment (sugaarK) [reply configs/default/HARC-HAKRCF405ICM.config:108]: This turns it on as a feature not as a define ? With out thia it won’t be be on when you flash the fc.. sorry was copying other targets

Comment (haslinghuis) [reply configs/default/HARC-HAKRCF405ICM.config:108]: #define USE_OSD does not enable feature OSD so this needs to stay.

**Comment by haslinghuis** [comment]
Would suggest HARC-HAKRCF405V2 or add the two #defines to the existing config.

File: configs/default/HARC-HAKRCF405ICM.config (hunk 1 above)
Comment (howels) [reply configs/default/HARC-HAKRCF405ICM.config:108]: Ok, thought this was defined centrally instead of in each board definition

**Comment by sugaarK** [comment]
All fixed up thanks @haslinghuis

**Review by KarateBrot — APPROVED** [review]

**Comment by haslinghuis** [comment]
Some fixes made:

Updated version to 4.4.0
Removed = from # dma
Removed `feature RX_SERIAL` as this is default (https://github.com/betaflight/betaflight/pull/11708)

**Review by blckmn — APPROVED** [review]

**Pull request merged**